lib/elixir/test/elixir/code_test.exs

```diff
@@ -74,7 +75,11 @@ defmodule CodeTest do
 
   test "eval_quoted/2 with a %Macro.Env{} struct as the second argument" do
     alias :lists, as: MyList
-    assert Code.eval_quoted(quote(do: MyList.flatten [[1, 2, 3]]), [], __ENV__) == {[1, 2, 3], []}
+
+    assert Code.eval_quoted(quote(do: MyList.flatten([[1, 2, 3]])), [], __ENV__) == {
+             [1, 2, 3],
+             []
+           }
```

Comment:
Please format this one and the others below like this:

```elixir
assert Code.eval_quoted(quote(do: MyList.flatten([[1, 2, 3]])), [], __ENV__) ==
           {[1, 2, 3], []}
```

Then you run the formatter again and the formatter should keep the formatting you adjusted.

Reply:
The code formatter keeps format the code as in the PR. I can’t do what you ask, or I misunderstood something :)

Reply:
It means your formatter is out of date. Please rebase/merge latest master, run `make clean compile` and then it should keep the formatting proposed.

Reply:
That’s what I’ve missed. `clean` to recompile the formatter. Thanks!

Reply:
Fixed now.